### PR TITLE
chgrp: add French translation

### PR DESCRIPTION
**Pull request by noraj**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.fr/common/chgrp.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Change récursivement le groupe propriétaire d'un répertoire et de son contenu :
+
+`chgrp -R {{groupe}} {{chemin/vers/répertoire}}`
+
+- Change le groupe propriétaire d'un lien symbolique :
+
+`chgrp -h {{groupe}} {{chemin/vers/lien_symbolique}}`
+
+- Modifier le groupe propriétaire d'un fichier/répertoire pour qu'il corresponde à un fichier de référence :
```
Comment (Nico385412) [inline pages.fr/common/chgrp.md:18]:
```suggestion
- Modifie le groupe propriétaire d'un fichier/répertoire pour qu'il corresponde à un fichier de référence :
```

Conjugate the verb :p

**Review by kbdharun — APPROVED** [review]
LGTM after addressing [Nico385412](https://github.com/Nico385412)'s comment.

File: pages.fr/common/chgrp.md (hunk 1 above)
Comment (noraj) [reply pages.fr/common/chgrp.md:18]: My bad.

**Review by blueskyson — APPROVED** [review]

**Review by Nico385412 — APPROVED** [review]

**Comment by kbdharun** [comment]
@Nico385412 we need to remove the "Co-authored by" part in the commit message during squash and merge. There is an ongoing PR regarding it in #8398.

**Comment by Nico385412** [comment]
> @Nico385412 we need to remove the "Co-authored by" part in the commit message during squash and merge. There is an ongoing PR regarding it in #8398.

Copy that ! i'll be more carefull next time !

**Pull request merged**